### ISSUE TITLE
Automatic restore of evicted nodes

### ISSUE THREAD

**Issue by kvaps** [piraeusdatastore/piraeus-operator#297]
If node come offline for 2 hours then return back it stays `EVICTED` even when linstor-satellite succefully started.
Shouldn't we recover such nodes automatically by piraeus-operrator?

**Comment by WanzenBug** [comment]
Yeah, you are right. I think restore got added a little later after the initial evict, so this was just missed.

**Comment by kvaps** [comment]
Automatic restoration still does not work to me.
@WanzenBug were you succeed in testing [this](https://github.com/piraeusdatastore/piraeus-operator/pull/298#issuecomment-1102664240)?

**Comment by WanzenBug** [comment]
Yes, I did test that successfully. But let me try again....

**Comment by kvaps** [comment]
@WanzenBug sorry to disturb, didn't you check that yet?
We have some users complaining on this issue

**Comment by WanzenBug** [comment]
Sorry, haven't checked it yet. Feel free to ping me again if I don't respond by next week

**Comment by WanzenBug** [comment]
@D1abloRUS This does sound like an issue that was recently fixed in LINSTOR, where the controller "forgot" to reconnect if the connection was dropped during the initial handshake. See https://github.com/LINBIT/linstor-server/commit/bbc27839ad69025b437696cd28598f1db3b80e77

Fixed version was just released, just need to release the new images for piraeus.
